app.js

```diff
@@ -1,25 +1,25 @@
-const express = require('express')
-const logger = require('morgan')
-const cors = require('cors')
+const express = require("express");
+const logger = require("morgan");
+const cors = require("cors");
 
-const contactsRouter = require('./routes/api/contacts')
+const contactsRouter = require("./routes/api/contacts");
 
-const app = express()
+const app = express();
 
-const formatsLogger = app.get('env') === 'development' ? 'dev' : 'short'
+const formatsLogger = app.get("env") === "development" ? "dev" : "short";
```

Comment:
I nazwa jest bardzo niekonkretna, bo czasownikowa, wiec powinna byc funkcja, ale wartosc to String. Proponuje `env`, czyli srodowisko, a wiec rzeczownik (to ma sens :))

Reply:
Albo `currentEnv`.

Reply:
Albo `loggerFormat`?

Reply:
Zmieniłem na currentEnv